src/utils/process.rs

```diff
@@ -426,19 +424,31 @@ where
     567  |       \_ less --RAW-CONTROL-CHARS --quit-if-one-screen
 
     */
-    #[cfg(test)]
-    {
-        info.find_sibling_process(my_pid, extract_args)
+
+    let pid_range = my_pid.saturating_sub(10)..my_pid.saturating_add(20);
+    for p in pid_range.clone() {
+        // Processes which were not refreshed do not exist for sysinfo, so by selectively
+        // letting it know about processes the `find_sibling..` function will only
+        // consider these.
+        info.refresh_process(p);
     }
-    #[cfg(not(test))]
-    {
-        None
+
+    match info.find_sibling_in_refreshed_processes(my_pid, &extract_args) {
+        None => {
+            #[cfg(test)]
+            {
+                info.refresh_processes();
+                info.find_sibling_in_refreshed_processes(my_pid, &extract_args)
```

Comment:
Out of interest can you explain why we make this recursive call in the tests? (Not blocking merge as this is a test-only code path).